CenterForOpenScience/osf.io
# [ENG-2120] Changing how caching works.

## Purpose

Improve caching for more frequent cache updated

## Changes

Adding a new caching method to make updates to storage cache. Removing usage of old caching method. Creating a management command to update all modified nodes.

## QA Notes

Workflows impacted: Storage Usage on Nodes

Does this change require a data migration? No

What is the level of risk?

Any permissions code touched?

Is this an additive or subtractive change, other?

How will QA verify? (UI, API, admin app or AdminAdminApp?) UI

How will this impact performance? This is an important question worth answering. It'd be good to test multiple users affecting many small files at the same time, compared to what it's like without this code.

## Documentation

N/A

## Side Effects

Might be some unintended WB side effects.

## DevNotes

Make sure to recreate your API container before testing.

## Ticket

https://openscience.atlassian.net/browse/ENG-2120
